lib/smart_app_launch/smart_stu1_suite.rb

```diff
@@ -0,0 +1,154 @@
+require 'tls_test_kit'
+
+require_relative 'version'
+require_relative 'discovery_stu1_group'
+require_relative 'standalone_launch_group'
+require_relative 'ehr_launch_group'
+require_relative 'openid_connect_group'
+require_relative 'token_refresh_group'
+
+module SMARTAppLaunch
+  class SMARTSTU1Suite < Inferno::TestSuite
+    id 'smart'
```

Comment:
The original suite was named `smart` so I left it as is.

Based on this comment https://github.com/inferno-framework/smart-app-launch-test-kit/pull/21#discussion_r896797837, I think this should stay `smart` since it represents an existing id.